cmd/tendermint/commands/debug/kill.go

```diff
@@ -33,7 +33,7 @@ $ tendermint debug kill 34255 /path/to/tm-debug.zip`,
 }
 
 func killCmdHandler(cmd *cobra.Command, args []string) error {
-	pid, err := strconv.ParseUint(args[0], 10, 64)
+	pid, err := strconv.ParseInt(args[0], 10, 32)
```

Comment:
probably should leave it as 64

Reply:
I've now been down the rabbit hole, and I think int32s are fine. 

I'm not super sure why we have this code for ourselves, but that's probably not a question for this PR

Reply:
https://elixir.bootlin.com/linux/latest/source/include/linux/threads.h#L33
the current limit is 2^30

but maybe it will require 64 bits in the later version.

Reply:
The size of `int` is chosen to be big enough to hold an index into addressable memory. On a 64-bit build, which is probably ~anything that will run Tendermint, `int` is likely to be 64-bits, so we should probably use 64 even if PIDs don't allow it.

Reply:
Yep, it's just a small fix. to close the warning.